tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java

```diff
@@ -45,7 +56,7 @@
     };
 
     // Thread-safe set implementation
-    private static final Set<Subscriber> SUBSCRIBERS = Sets.newConcurrentHashSet();
+    private static final Set<SpanObserver> observers = Sets.newConcurrentHashSet();
```

Comment:
well, this isn't a CONSTANT, it's a static final thing that changes over time, similar to `log`